decidim-admin/app/packs/src/decidim/admin/global_moderations.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-invalid-this */
```

Comment:
Being a new JS file, please Update the syntax to vanilla JS.

Reply:
Done